README.md

```diff
@@ -85,7 +86,7 @@ EOF
 Once the Operator is up and running, you can request instances of Authorino by creating `Authorino` CRs. E.g.:
 
 ```sh
-kubectl -n myapi apply -f -<<EOF
+kubectl -n authorino-operator apply -f -<<EOF
```

Comment:
This will not work. The idea here is to highlight that Authorino can be deployed as well as a dedicated instance within each application namespace.

Authorino deployed to the `authorino-operator` ns makes sense when using Authorino in cluster-wide mode, which is not the case of the exemplified CR.

Reply:
hmm, that's fair. I'm happy to revert it back to `myapi` if that's the preference but I would prefer if the namespace exists 🤔

Any thoughts on using the `default` namespace ?

Reply:
`default` is fine. In fact, it would match most of the tutorials.

Reply:
Cool, I've pushed the change to use `default` namespace instaad 👍